src/AutoRest.CSharp/MgmtTest/AutoRest/MgmtTestTarget.cs

```diff
@@ -14,6 +17,11 @@ internal class MgmtTestTarget
     {
         public static void Execute(GeneratedCodeWorkspace project, CodeModel codeModel, SourceInputModel? sourceInputModel)
         {
+            if (codeModel.TestModel is null)
+            {
+                throw new NotSupportedException("The codeModel.TestModel is required for test Generation!");
+            }
```

Comment:
Will this happen? I believe we checked the same thing when executing this `Execute` method

Reply:
Yes, currently it won't happen, adding it because 1) it can tell the compiler not to check null for codeModel.TestModel, and 2) it may give some description when outside code changes